packages/app-project/pages/404.js

```diff
@@ -47,7 +49,7 @@ export default function Error404({
   const { t } = useTranslation('components')
 
   return (
-    <>
+    <ThemeContext.Extend value={theme404}>
       <ZooHeaderWrapper locale={locale} />
       <ContainerBox
```

Comment:
I can't directly comment on it, but this ContainerBox has a `background` prop that's being overwritten by `style={{ backgroundColor }}`. I recommend removing `background` if the color is declared in the style object instead.

Reply:
Also the ContainerBox and its Overlay are set at 50vh. The Figma designs look like 100vh - ZooHeader height though. Is the difference from the designs intentional?

Reply:
I can't remember when it was discussed yet the reason it's something smaller than the whole viewport is so that the footer is visible. When it was originally posed Sean didn't create a follow up item to change it. I think we can ask @seanmiller26 to clarify.. Specifically, how much of the viewport should the 404 background image take up compared to the ZooHeader and ZooFooter?

- XL page it takes up the whole viewport minus the header
- L page it takes up ~60%
- M page takes up whole viewport
- S page takes up whole viewport

Reply:
@kieftrav Unfortunately Figma doesn't support relative units like vh, vw, vmin, vmax, etc, so calculating all of that manually is a bit of a chore. If I can ever specify these values in the future I'll be sure to do that.

Not sure what discussion mentioned the footer being visible, and I don't disagree, but let's go ahead and try to match the Figma design a little closer. Can we try a 80vh to split the difference?